Tests/test_KDTree.py

```diff
@@ -20,7 +20,7 @@
     raise MissingExternalDependencyError(
             "C module in Bio.KDTree not compiled")
 
-from Bio.KDTree import KDTree, _CKDTree
+from Bio.KDTree.KDTree import KDTree
```

Comment:
Does this mean these changes altered how end users would import the ``KDTree`` object?

Reply:
No it won't; the class structure is unchanged everywhere. Previously the test was accessing the underlying C object directly. I have no idea why it was done that way, but it didn't matter then. Now, some code can only be access through the Python object.

Reply:
I understand why we don't care about ``_CKDTree`` - it was private anyway.

It is this change bothers me:

```
from Bio.KDTree import KDTree
```

vs

```
from Bio.KDTree.KDTree import KDTree
```

Are there now three things called ``KDTree`` (module, sub-module, class?).

Reply:
There were always three things called KDTree - which is why I don't like the current code design. The three are
`Bio/KDTree/__init__.py`, `Bio/KDTree/KDTree.py`, and the class `KDTree` inside `Bio/KDTree/KDTree.py`.

Wait, there is one more: the class `KDTree` inside `Bio/KDTree/KDTreemodule.c`. So four KDTrees.

Reply:
That's worse than I realised. I can see why PEP8 recommends modules in lower case and classes in title case - that would have helped vastly. We would want to follow that in any new module structure for this stuff.